dist/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
- [ ] In the `readme.md` there are some necessary sections that are lacking. Kindly follow [this template](https://github.com/microverseinc/readme-template) and include them.